Gemfile

```diff
@@ -17,15 +17,21 @@ gem 'pronto-sorbet', require: false
 
 # Rubocop Extentions
 # see https://docs.rubocop.org/rubocop/extensions.html
-gem 'rubocop-performance', require: false
-gem 'rubocop-rails', require: false
-gem 'rubocop-rspec', require: false
-gem 'rubocop-thread_safety', require: false
-gem 'rubocop-require_tools', require: false
-gem 'rubocop-i18n', require: false
-gem 'rubocop-sequel', require: false
-# gem 'cookstyle', require: false # don't support latest rubocop
-gem 'rubocop-rake', require: false
-
-gem 'rubocop-md', require: false
-gem 'rubocop-minitest', require: false
+## Rubocop Official Extentions
+gem 'rubocop-performance', '~> 1', require: false
+gem 'rubocop-rails', '~> 2', require: false
+gem 'rubocop-rspec', '~> 2', require: false
+gem 'rubocop-minitest', '~> 0', require: false
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `rubocop-minitest` should appear before `rubocop-rspec`.